src/kvstore/Part.h

```diff
@@ -23,9 +23,12 @@ class Part : public raftex::RaftPart {
          const std::string& walPath,
          KVEngine* engine,
          std::shared_ptr<folly::IOThreadPoolExecutor> pool,
-         std::shared_ptr<thread::GenericThreadPool> workers);
+         std::shared_ptr<thread::GenericThreadPool> workers,
+         wal::BufferFlusher* flusher);
 
-    virtual ~Part() = default;
+    virtual ~Part() {
+        LOG(INFO) << idStr_ << "~Part()";
```

Comment:
space

Reply:
There is already on space left in idStr_